keyboards/diverge3/config.h

```diff
@@ -51,50 +53,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* COL2ROW, ROW2COL, or CUSTOM_MATRIX */
 #define DIODE_DIRECTION ROW2COL
 
-// #define BACKLIGHT_PIN B7
+// #define BACKLIGHT_PIN C6
 // #define BACKLIGHT_BREATHING
 // #define BACKLIGHT_LEVELS 3
 
 
 /* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
 #define DEBOUNCING_DELAY 5
 
-/* define if matrix has ghost (lacks anti-ghosting diodes) */
-//#define MATRIX_HAS_GHOST
-
-/* number of backlight levels */
-
-/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
-#define LOCKING_SUPPORT_ENABLE
```

Comment:
Any reason for removing the code here?

Reply:
My mistake, I didn't think it applied, so I'm adding it back.